src/lib/Behat/Page/ContentUpdateItemPage.php

```diff
@@ -185,4 +188,36 @@ public function verifyFieldCannotBeEditedDueToLimitation(string $fieldName)
             ->getLocator('fieldGroupNthField')->getSelector(), $activeSections->single()->getAttribute('data-anchor-target-section-id'), $this->getFieldPosition($fieldName)));
         $this->getHTMLPage()->find($fieldLocator)->assert()->hasClass('ibexa-field-edit--disabled');
     }
+
+    public function verifyAutosaveNotificationIsDisplayed(): void
+    {
+        $this->getHTMLPage()
+            ->find($this->getLocator('autosaveIsOnInfo'))
+            ->assert()->textContains('Autosave is on');
+    }
+
+    public function verifyAutosaveDraftIsSavedNotificationIsDisplayed(): void
+    {
+        $iteration_count = 20;
+
+        while ($this->isAutosaveDraftSavedNotificationVisible() == false && $iteration_count > 0) {
+            usleep(500000);
+            --$iteration_count;
```

Comment:
Currently this loop has no failure mode.

It either:
1) Exists early, if Content Item has been autosaved
2) Exists after 10 seconds, if the Content Item has not been autosaved

In case of `2` there's no failure - the test simply takes longer, but will still pass.

We could do something like this:
```
$iteration_count = 20;

while ($iteration_count > 0) {
    if ($this->isAutosaveDraftSavedNotificationVisible()) {
        return;
    }
    usleep(500000);
    --$iteration_count;
}

Assert::fail('Draft has not been autosaved for 10 seconds');
```

This way we get notified when the correct message is not displayed in the expected time.